server/routes/activities/record-attendance/recordAttendanceRequests.ts

```diff
@@ -3,6 +3,13 @@ export type SessionCancellationRequest = {
   comment?: string
 }
 
+export enum AttendActivityMode {
+  MULTIPLE = 'MULTIPLE',
+  SINGLE = 'SINGLE',
+}
+
 export type RecordAttendanceRequests = {
-  sessionCancellation: SessionCancellationRequest
+  sessionCancellation?: SessionCancellationRequest
+  mode?: AttendActivityMode
```

Comment:
Allows the user journey to return to the correct view after edit / non-attend etc